product_weight/wizard/product_weight_update_view.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+    <record id="view_product_weight_update" model="ir.ui.view">
+      <field name="name">Update Product Weight</field>
+      <field name="model">product.weight.update</field>
+      <field name="arch" type="xml">
+	<form string="Update Product Weight" version="8.0">
```

Comment:
You can remove `version="8.0"` version was only an attribute for transition for new 7.0 web client